ui/src/app/pipeline-details/components/edit/quickedit.component.html

```diff
@@ -16,52 +16,75 @@
   ~
   -->
 
-<sp-basic-nav-tabs [spNavigationItems]="tabs"
-                   [activeLink]="'quick-edit'"
-                   [showBackLink]="true"
-                   [backLinkTarget]="['pipelines']">
-
+<sp-basic-nav-tabs
+    [spNavigationItems]="tabs"
+    [activeLink]="'sp-quick-edit'"
```

Comment:
is it correct to adapt this reference as well?

Reply:
No, this should not be changed.
This is related to the route and not the component.
The route is registered in the module (see [1])

[1] https://github.com/apache/streampipes/blob/dev/ui/src/app/pipelines/pipelines.module.ts

Reply:
Alright, then I'll revert it 🙂